GDJS/Runtime/spriteruntimeobject.js

```diff
@@ -187,13 +292,14 @@ gdjs.SpriteRuntimeObject = function(runtimeScene, objectData)
      * call `this._updateAnimationFrame()`.
      * Can be null, so ensure that this case is handled properly.
      *
-     * @type gdjs.SpriteAnimationFrame
+     * @type {gdjs.SpriteAnimationFrame}
      */
     this._animationFrame = null;
 
     if (this._renderer)
         gdjs.SpriteRuntimeObjectRenderer.call(this._renderer, this, runtimeScene);
     else
+        /** @type {gdjs.SpriteRuntimeObjectRenderer} */
```

Comment:
Can you confirm that autocompletion works with these types in VSCode?
gdjs.SpriteRuntimeObjectRenderer is not really a type that is existing, it's either `gdjs.SpriteRuntimeObjectCocosRenderer` or `gdjs.SpriteRuntimeObjectPixiRenderer` according to which renderer you're using.

Typescript was properly inferring `gdjs.SpriteRuntimeObjectRenderer = gdjs.SpriteRuntimeObjectCocosRenderer | gdjs.SpriteRuntimeObjectPixiRenderer` in VSCode, I'm afraid this could break it?

Reply:
There is no autocomplete, but if we add a jsdoc for gdjs.SpriteRuntimeObjectRenderer that describes it's properties it should work. In the meantime it still indicates what class it is, so it's still useful

Reply:
We will add this later in another pr

Reply:
Yes I think we can keep these renderer typings in another PR.